prdoc/pr_7437.prdoc

```diff
@@ -0,0 +1,10 @@
+title: Update SCALE codec indices
```

Comment:
To be more exact it is a removal of inner dead code.

Reply:
Ah, got confused with the MRs. We need more CI to copy these files...

Reply:
@ggwpez so is the prdoc needed ? Even if we're just removing dead code ? If so I can add it to the other backports as well